src/native/fs/fs_napi.cpp

```diff
@@ -65,6 +72,69 @@ api(const Napi::CallbackInfo& info)
     return promise;
 }
 
+void
+set_stat_res(Napi::Object res, Napi::Env env, struct stat _stat_res)
```

Comment:
send struct by reference, and rename without leading `_`
```suggestion
set_stat_res(Napi::Object res, Napi::Env env, struct stat &stat_res)
```